src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingService.java

```diff
@@ -429,9 +432,19 @@ public IBinder onBind(Intent intent) {
      */
     @Override
     public void onDestroy() {
-        super.onDestroy();
-        Log.i(TAG, "Shutting down service");
-        shutdown();
+//        super.onDestroy();    Call of super not required -> https://developer.android.com/guide/components/services.html
```

Comment:
Okay thats cool. Yeah, onDestroy() got somewhat complicated with these changes, so you should add a comment saying what it does in which case (and probably mention that onDestroy is not called and not required). onPostExecute() doesn't look so complicated, but more documentation surely won't hurt.